types/aws-lambda/trigger/iot-authorizer.d.ts

```diff
@@ -0,0 +1,48 @@
+import {Handler} from "../handler";
+import {PolicyDocument} from "./api-gateway-authorizer";
+
+export type IoTProtocolType = "http" | "mqtt" | "tls";
+
+export type IoTCustomAuthorizerHandler = Handler<IoTCustomAuthorizerEvent, IoTCustomAuthorizerResult>;
```

Comment:
Technically there should also be a Callback type alias here as well for consistency, but honestly they should all be marked deprecated at this point, even if the callback parameter itself isn't.

Anyone else have an opinion here?